expression/builtin_time_vec_test.go

```diff
@@ -52,11 +52,13 @@ var vecBuiltinTimeCases = map[string][]vecExprBenchCase{
 	ast.CurrentDate: {
 		{retEvalType: types.ETDatetime},
 	},
-	ast.MakeDate:   {},
-	ast.MakeTime:   {},
-	ast.PeriodAdd:  {},
-	ast.PeriodDiff: {},
-	ast.Quarter:    {},
+	ast.MakeDate:  {},
+	ast.MakeTime:  {},
+	ast.PeriodAdd: {},
+	ast.PeriodDiff: {
+		{retEvalType: types.ETInt, childrenTypes: []types.EvalType{types.ETInt, types.ETInt}, geners: []dataGenerator{&rangeInt64Gener{1, 13}, &rangeInt64Gener{1, 13}}},
```

Comment:
Could you make a new `gener` in builtin_time_vec_test.go?

Reply:
Yes, I add `periodGener`